src/components/VegaChart/index.js

```diff
@@ -23,7 +23,7 @@ const VegaChart = ( { spec } ) => {
 	}, [ container, id, spec ] );
 
 	return (
-		<div ref={ container }>
+		<div className={ 'vega-chart' } ref={ container }>
```

Comment:
```suggestion
		<div className="vega-chart" ref={ container }>
```
Variable syntax not needed with a string literal.